lms/djangoapps/courseware/views.py

```diff
@@ -417,6 +417,7 @@ def _index_bulk_op(request, course_key, chapter, section, position):
             'course': course,
             'init': '',
             'fragment': Fragment(),
+            'course_ga_tag': course.course_google_analytics_tag,
```

Comment:
Similarly, let's expose this elsewhere,
but augment the context with a `context.update` call.